src/runtime-rs/crates/resource/src/rootfs/mod.rs

```diff
@@ -4,24 +4,30 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+mod nydus_rootfs;
 mod share_fs_rootfs;
 
-use std::{sync::Arc, vec::Vec};
-
+use agent::Storage;
 use anyhow::{anyhow, Context, Result};
 use async_trait::async_trait;
+use hypervisor::Hypervisor;
 use kata_types::mount::Mount;
 use nix::sys::stat::{self, SFlag};
+use std::{sync::Arc, vec::Vec};
 use tokio::sync::RwLock;
 
 use crate::share_fs::ShareFs;
 
-const ROOTFS: &str = "rootfs";
+use self::nydus_rootfs::NYDUS_ROOTFS_TYPE;
 
+const ROOTFS: &str = "rootfs";
+const HYBRID_ROOTFS_WRITABLE_LAYER_LOWER_DIR: &str = "rootfs_lower";
+const TYPE_OVERLAY_FS: &str = "overlay";
```

Comment:
nit: use `const OVERLAY_FS` instead of `const TYPE_OVERLAY_FS` to be consistent with `const ROOTFS`

Reply:
`TYPE_OVERLAY_FS` is representing the `fs_type`, `ROOTFS` is used as a prefix.